test/test.js

```diff
@@ -35,7 +40,10 @@ test('equals to sample', async t => {
 
 test('check internal structure', async t => {
   // Create deck as in previous example
-  const apkg = new AnkiExport('deck-name');
+  const apkg = new AnkiExport('deck-name', {
+    template,
+    DbClass: sql.Database
```

Comment:
> Can't understand how it make tests pass

Because `AnkiExport` is (still) exported from `index.js` instead of `exporter.js`. See the second paragraph from my message above https://github.com/ewnd9/anki-apkg-export/pull/8#discussion_r91722957